src/structures/buffers.rs

```diff
@@ -92,6 +92,14 @@ pub mod data {
     buffer_type!(Data, 64, TPM2B_DATA);
 }
 
+pub mod sensitive_data {
+    buffer_type!(SensitiveData, 256, TPM2B_SENSITIVE_DATA);
```

Comment:
Nice!

The reason that SensitiveData type was alone in ```src/algorithm/structures/sensitive/data.rs``` was because when I did a rather big refactoring a while back. I used the chapters in TPM Specification (Structures) as template for how to structure things, so because the sensitive data structure is specified in *Algorithm Parameters and Structures* chapter it ended up with the path it did (now that I look at it should actually have been 
```src/algorithm/structures/symmetric/sensitive/data.rs```).

But maybe that was not best way to do things ... perhaps it would have made more sense to do
 --> TPM2B - folder
 --> TPMI -folder
 --> TPML -folder

and so on ...


If the latter makes more sense then I think we should make an effort to structure things that way instead

Reply:
Right, though I was under the assumption that `buffers.rs` referred to `TPM2B_`-types, and `src/structures/lists/*` would be for `TPML_` structures.
So basically based it on the type prefix.
Honestly, if you prefer it in `src/algorithm/structures/symmetric/sensitive/data.rs`, that's fine, and I don't have an opinion about that, other than that I'd prefer `src/structures/algorithm/...` in that case, since it's still in the `Structures` part of the spec.
But I do thing that having it in `buffers.rs` makes sense as it's a `TPM2B_` type.

Reply:
I have no strong opinion regarding this.

Right now I am leaning towards that it maybe would be better idea to structure things around the prefix instead.

Reply:
So, what would you prefer? Since I'd say that `buffers.rs` is the place for `TPM2B_` objects, or am I misunderstanding what you mean?

Reply:
No, I think you are right. I think it is more logical to structure things around the prefix. So placing TPM2B in buffer.rs is a good idea. I just mean that we need to be consistent with this approach. Maybe do another refactoring soon in order to ensure this structure.

Reply:
I'm pro prefix-based placement as well - I noticed the weird placement of `SensitiveData` before but figured it was just a weird location in the spec